static/install.html

```diff
@@ -191,7 +198,7 @@ <h2 id="obtaining-factory-images">
             <a href="https://www.reddit.com/r/GrapheneOS/comments/c7gb3f/grapheneos_factory_images_are_now_signed_with/esewpm9">the /u/GrapheneOS
             Reddit account</a> and <a href="https://github.com/GrapheneOS/releases.grapheneos.org/blob/master/static/factory.pub">is available on GitHub</a>.
             When the current signing key is replaced, the new key will be signed with it.</p>
-            <p>Download the factory images for the device from <a href="/releases">the releases
+            <p>Download both factory images(*.zip and *.zip.sig) for the device from <a href="/releases">the releases
```

Comment:
I can see what you're trying to do here but the implementation of the change doesn't make sense. The signature isn't another set of factory images. I'll change this and you can just omit the change.